app/views/layouts/application.html.erb

```diff
@@ -40,6 +14,46 @@
 </head>
 <body>
 
+
+
```

Comment:
Here I added the markup for the example autocomplete.  You'll probably want to remove this, but I used this to test if it actually works.